advanced_functionality/autogluon-multimodal-containers/ag_model.py

```diff
@@ -0,0 +1,89 @@
+from sagemaker.estimator import Framework
+from sagemaker.predictor import Predictor
+from sagemaker.mxnet import MXNetModel
+from sagemaker.mxnet.model import MXNetPredictor
+from sagemaker import utils
+from sagemaker import image_uris
+from sagemaker.serializers import CSVSerializer
+from sagemaker.deserializers import StringDeserializer
+
+
+class AutoGluonTraining(Framework):
```

Comment:
There are some common logic between Tabular and MM, is it recommended by Sagemaker team to replicate these codes for each example or we can extract out the common logics for reuse

Reply:
This is not covered in the [Contribution Guidelines](https://github.com/aws/amazon-sagemaker-examples/blob/main/CONTRIBUTING.md). I can make a comment on my official PR regarding this when I submit it and see what the repo maintainers have to say.